src/ex1/script.js

```diff
@@ -0,0 +1,141 @@
+let taskId = 0;
+let tasksCnt = 0;
+let isAddTaskClickable = false;
+const addTaskInput = document.querySelector(".add-task-input");
+const addTaskBtn = document.querySelector(".add-task-btn");
+
+// Add task will be clickable only for a non empty task
+addTaskInput.addEventListener("input", () => {
+  if (addTaskInput.value && !isAddTaskClickable) {
+    addTaskBtn.classList.add("valid-task-btn");
+    isAddTaskClickable = true;
+  } else if (!addTaskInput.value) {
+    addTaskBtn.classList.remove("valid-task-btn");
+    isAddTaskClickable = false;
+  }
+});
+
+// Add task
+addTaskBtn.addEventListener("click", (event) => onAddTask(event));
```

Comment:
```suggestion
addTaskBtn.addEventListener("click", onAddTask);
```
should be enough ^

Reply:
Agree :) This line was changed in Ex2, but I found some other lines where I was doing the same and changed them, though there are some places where I needed an arrow function for "this" issues.